src/components/Contact/Contact.module.scss

```diff
@@ -0,0 +1,93 @@
+.container {
+    max-width: 1440px;
+    border-bottom: 1px solid rgba(255, 255, 255, 0.2);
+    margin-top: 100px;
+    margin-bottom: 100px;
+}
+
+.contact {
+&__textWrap {
+    display: block;
+    justify-content: center;
+    text-align: center;
+    width: 600px;
+    margin: auto; 
+    margin-bottom: 50px;
+}
+
+&__title {
+    font-family: Banny;
+    font-size: 36px;
+    line-height: 48px;
```

Comment:
in cases, when you change at least font-family and font-size, use font property

https://developer.mozilla.org/ru/docs/Web/CSS/font